File: src/main/java/com/example/dto/AccountDto.java
package com.example.dto;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
@Builder
@EqualsAndHashCode

@NoArgsConstructor
public class AccountDto {


    private String id;
    private BigDecimal balance ;
    private LocalDateTime creatitionTime;
    private AccountCustomerDto accountCustomerDto;
    private List<TransactionDto> transactionDtoSet;

}

File: src/main/java/com/example/dto/AccountDto.kt
package com.example.dto

import java.math.BigDecimal
import java.time.LocalDateTime



data class AccountDto (
     val id: String? ,
     val balance: BigDecimal?,
     val creatitionTime: LocalDateTime? ,
     val accountCustomerDto: AccountCustomerDto? ,
     val transactionDtoSet: List<TransactionDto>?
)